src/python/WMCore/WorkQueue/WMBSHelper.py

```diff
@@ -326,6 +326,12 @@ def _createSubscriptionsInWMBS(self, task, fileset, alternativeFilesetClose=Fals
                 mergedOutputFileset = None
 
                 for childTask in task.childTaskIterator():
+                    # # NOTE: The two lines bellow will propagate the upper level task's site lists to the child tasks in wmbs only.
```

Comment:
Should we remove this?

Reply:
yes, the whole change to WMBSHelper will be removed while squashing the commits.

Reply:
Thanks for the review @amaltaro I just addressed your comment, could you please take another look.